doc/src/modules/physics/mechanics/examples/four_bar_linkage_example.rst

```diff
@@ -28,16 +27,25 @@ lengths and density of the links. ::
    >>> q1, q2, q3, u1, u2, u3 = dynamicsymbols('q1:4, u1:4')
    >>> l1, l2, l3, l4, rho = symbols('l1:5, rho')
 
-With all symbols defined, we can now define the bodies. ::
+With all symbols defined, we can now define the bodies and initialize our
+instance of :class:`~.System`. ::
 
    >>> N = ReferenceFrame('N')
-   >>> inertias = [inertia(N, 0, 0, rho * l ** 3 / 12) for l in (l1, l2, l3, l4)]
-   >>> link1 = Body('Link1', frame=N, mass=rho * l1, central_inertia=inertias[0])
-   >>> link2 = Body('Link2', mass=rho * l2, central_inertia=inertias[1])
-   >>> link3 = Body('Link3', mass=rho * l3, central_inertia=inertias[2])
-   >>> link4 = Body('Link4', mass=rho * l4, central_inertia=inertias[3])
+   >>> mass_centers = [Point(f'mc{i}') for i in range(1, 5)]
+   >>> inertias = [Inertia.from_inertia_scalars(P, N, 0, 0, rho * l ** 3 / 12)
```

Comment:
Minor style note. We typically do not have spaces around multiple, divide or power in sympy. It would be best if this was consistent with sympy's style.

Reply:
Addressed for this file in 72ba1293f3f5986703a481248a7588c9d54a31b0

Reply:
SymPy "mostly follows PEP8", this is one of the things that we typically don't follow.

Reply:
I'll try to keep it in mind in the future. Though I am used to having my IDE do all formatting for me.